macros/unpack/get_column_values.sql

```diff
@@ -0,0 +1,46 @@
+{%- macro get_column_values(node_type) -%}
+    {{ return(adapter.dispatch('get_column_values', 'dbt_project_evaluator')(node_type)) }}
+{%- endmacro -%}
+
+{%- macro default__get_column_values(node_type) -%}
+
+    {%- if execute -%}
+        {%- if node_type == 'nodes' %}
+            {% set nodes_list = graph.nodes.values() %}   
+        {%- elif node_type == 'sources' -%}
+            {% set nodes_list = graph.sources.values() %}
+        {%- else -%}
+            {{ exceptions.raise_compiler_error("node_type needs to be either nodes or sources, got " ~ node_type) }}
+        {% endif -%}
+
+        {%- set values = [] -%}
+
+        {%- for node in nodes_list -%}
+            {%- for column in node.columns.values() -%}
+
+                {%- set values_line  = 
+                    [
+                        wrap_string_with_quotes(node.unique_id),
+                        wrap_string_with_quotes(node.name),
+                        wrap_string_with_quotes(node.resource_type),
+                        wrap_string_with_quotes(node.original_file_path | replace("\\","\\\\")),
+                        wrap_string_with_quotes(node.database),
+                        wrap_string_with_quotes(node.schema),
+                        wrap_string_with_quotes(node.package_name),
+                        wrap_string_with_quotes(node.alias),
```

Comment:
Could we remove those and instead join with the node model? We would just need the `unique_id` to be able to perform this join.

This table potentially having many rows in it, whatever we can do to make is as "small" as possible will help.